src/main/kotlin/no/nav/syfo/client/SyfohelsenettproxyClient.kt

```diff
@@ -40,6 +41,7 @@ class SyfohelsenettproxyClient(
         }
 
         val httpResponse = httpStatement.execute()
+        log.info("Fikk response status: ${httpResponse.status}, String: ${httpResponse.call.response.receive<String>()}")
```

Comment:
Er det fare for at dette logger noe som ikke bør logges i produksjon?

Reply:
Søren! Ja, det er nok det 😢

Reply:
Jeg tok den bort nå, jeg overså den når jeg fjernet debug-logging 🤷‍♂️